src/memprof_postjs.js

```diff
@@ -0,0 +1,2 @@
+// Will contain memory leak checks.
```

Comment:
We could just kill this file? In the future if it becomes necessary, add it in then.